integration_tests/src/main/python/window_function_test.py

```diff
@@ -383,6 +383,47 @@ def test_window_aggs_for_rows(data_gen, batch_size):
         conf = conf)
 
 
+@ignore_order(local=True)
+@pytest.mark.parametrize('batch_size', ['1000', '1g'], ids=idfn)
+@pytest.mark.parametrize('data_gen', [
+    [('grp', RepeatSeqGen(int_gen, length=20)),  # Grouping column.
+     ('ord', StringGen(nullable=True)),          # Order-by column (STRING).
```

Comment:
nit: if we get unlucky then we can have an ambiguous order. It would be nice if we could do something like LongRangeGen bit for strings, or perhaps just cast the result of LongRangeGen to a string before doing the order by on it.

Reply:
Good catch. This should save heartache later on.